Build/Projects/MGCB.definition

```diff
@@ -10,6 +10,8 @@
     <Reference Include="System.Xml" />
     <Reference Include="MonoGame.Framework" />
     <Reference Include="MonoGame.Framework.Content.Pipeline" />
+
+    <Reference Include="MGCB.References" />
```

Comment:
Refer to my other comment, but basically by removing platforms you prevent the games consuming MonoGame from having platform specific code because they are forced to use DesktopGL as their platform.